CMakeLists.txt

```diff
@@ -70,6 +70,10 @@ if((${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.8) AND (NOT ABSL_PROPAGATE_CXX_STD)
   message(WARNING "A future Abseil release will default ABSL_PROPAGATE_CXX_STD to ON for CMake 3.8 and up. We recommend enabling this option to ensure your project still builds correctly.")
 endif()
 
+option(ABSL_USE_SYSTEM_INCLUDES
+  "Mark C++ include directories with `SYSTEM` to prevent warnings from headers when using Abseil with incompatible warning flags"
```

Comment:
In my opinion this sentence hard to parse. What do you think about something like "Silence warnings in Abseil headers by marking them as `SYSTEM` includes"?

Reply:
Fair enough, I didn't think very much about the message. Updated in new commit